Alteration of the following patch to force fork mode iso spawn

From f093da4a3a457d539e5682ccecdf91f254addd8c Mon Sep 17 00:00:00 2001
From: Zac Medico <zmedico@gentoo.org>
Date: Thu, 3 Dec 2020 21:37:39 -0800
Subject: [PATCH] runTests.py: multiprocessing.set_start_method('spawn') for
 debugging bug 758230

Force fork mode because spawn mode requires all argument to be
pickle-able, which currently is not the case.  This is an issue because
Python 3.8 changed the default from fork to spawn on macOS.

diff --git a/bin/emerge b/bin/emerge
index 8f1db61a6..3731a9081 100755
--- a/bin/emerge
+++ b/bin/emerge
@@ -2,6 +2,7 @@
 # Copyright 2006-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+import multiprocessing
 import platform
 import signal
 import sys
@@ -44,6 +45,7 @@ try:
 	from _emerge.main import emerge_main
 
 	if __name__ == "__main__":
+		multiprocessing.set_start_method('fork')
 		from portage.exception import IsADirectory, ParseError, \
 				PermissionDenied
 		portage.process.sanitize_fds()
